### ISSUE TITLE
Apache 2.4 support

### ISSUE THREAD

**Issue by teadur** [mod-epp/mod-epp#2]
Is there someone working on getting mod-epp work with apache 2.4 ?

**Comment by gbxyz** [comment]
Have you tried using `mod_epp` with Apache 2.4? Do you get a compile error?

No-one's actively working on it, but we (@CentralNic) will be migrating to 2.4 at some point, so support for 2.4 (if it doesn't already work) may come along then.

**Comment by jonasbn** [comment]
I have tried, but without success. I cannot remember the exact issues or error messages. I will see if I can dig up some more information.

**Comment by teadur** [comment]
@gbxyz Any progress with the 2.4 update @centralnic ?

Our first attempts on upgrading mod-epp to 2.4 ended in negative results:
https://github.com/internetee/registry/issues/1015#issuecomment-482451175

**Comment by gbxyz** [comment]
No progress yet, but it's on our internal roadmap for this year.

**Comment by jonasbn** [comment]
We have decided to roll our own and are migrating away from **mod_epp**. The architecture we rely on resembles the one imposed by **mod_epp**, so the spirit still lives in our platform.

It has been a good long run...

**Comment by teadur** [comment]
We are tryng to write our own from scratch aswell, once it seems stable/good enough i will promote it here.

**Comment by artur-intech** [comment]
> We are tryng to write our own from scratch aswell, once it seems stable/good enough i will promote it here.

https://github.com/internetee/epp_proxy

**Comment by registrio** [comment]
We use mod_epp with Apache 2.4.29 and is ok, but we test on newer versions, it gets compile errors.

**Comment by thinkpaduser** [comment]
If someone is still interested, we written a working patch for this module.
This is httpd version we using mod_epp with: httpd-2.4.6-80.el7.centos.1.x86_64

**Comment by registrio** [comment]
> If someone is still interested, we written a working patch for this module.
> This is httpd version we using mod_epp with: httpd-2.4.6-80.el7.centos.1.x86_64

Can you share it on gist? Thank you.

**Comment by thinkpaduser** [comment]
> Can you share it on gist? Thank you.

Yes, but I did some review, there are much code differences made by some people. I honestly don't know which are necessary.

**Comment by registrio** [comment]
> > Can you share it on gist? Thank you.
> 
> Yes, but I did some review, there are much code differences made by some people. I honestly don't know which are necessary.

Share it and I'll review and let you know.
